images/macos/helpers/Xcode.Helpers.psm1

```diff
@@ -121,4 +121,29 @@ function Get-XcodePairsList {
         $result += "$watchName $phoneName"
     }
     return $result
+}
+
+function Invoke-XCVersion {
+    param(
+        [Parameter(Mandatory)]
+        [string] $Command,
+        [Parameter()]
+        [int] $RetryAttempts = 7,
+        [Parameter()]
+        [int] $PauseDuration = 1
+    )
+
+    do {
+        Write-Host "Current attempt: [$RetryAttempts]"
```

Comment:
Let's write command that we are trying to run to logs too

Reply:
Done